pkg/contexts/ocm/transfer/transfer.go

```diff
@@ -215,6 +215,9 @@ func transferVersion(printer common.Printer, log logging.Logger, state WalkingSt
 }
 
 func CopyVersion(printer common.Printer, log logging.Logger, hist common.History, src ocm.ComponentVersionAccess, t ocm.ComponentVersionAccess, handler TransferHandler) (rerr error) {
+	if printer == nil {
+		printer = common.NonePrinter
+	}
```

Comment:
@mandelsoft is this the fix for https://github.com/open-component-model/ocm/issues/708 ???